spec/models/person_spec.rb

```diff
@@ -24,6 +24,20 @@
     end
   end
 
+  describe 'email validation' do
+    let(:contact_method) { build :contact_method, name: 'Email', field: 'email' }
+    subject(:person) { build :person, preferred_contact_method: contact_method, email: 'test@missingtld' }
+
+    context 'when the email field is not valid' do
+      it { is_expected.not_to be_valid }
+
+      it 'generates an error on the correct field' do
+        person.valid?
```

Comment:
Do you think these tests would be better as tests on the validator itself instead of on the model?

Reply:
Done :+1: